README.md

```diff
@@ -59,7 +59,7 @@ The following options are available:
 | `faviconUrl`            | String  | `null`                     | The URL to the favicon.                                                                                                                                        |
 | `siteTitle`             | String  | `"{{ site.title.value }}"` | The site title.                                                                                                                                                |
 | `siteUrl`               | String  | `"{{ site.url }}"`         | The site URL.                                                                                                                                                  |
-| `titleSeparator`        | String  | `"-"`                      | The title separator between the page's and the site title. Only applies if no `titleContentKey` is set the target field value is empty. Also supports queries. |
```

Comment:
I think `page's` is better, as it clearly indicated a single page-site-relationship is meant.